waiter/src/waiter/auth/authentication.clj

```diff
@@ -21,10 +21,22 @@
             [waiter.cookie-support :as cookie-support]
             [waiter.middleware :as middleware]
             [waiter.status-codes :refer :all]
-            [waiter.util.utils :as utils]))
+            [waiter.util.ring-utils :as ru]
+            [waiter.util.utils :as utils])
+  (:import (java.net URI)))
+
+(def ^:const AUTH-COOKIE-EXPIRES-AT "x-auth-expires-at")
 
 (def ^:const AUTH-COOKIE-NAME "x-waiter-auth")
 
+(def ^:const auth-expires-at-uri "/.well-known/auth/expires-at")
+
+(def ^:const auth-keep-alive-uri "/.well-known/auth/keep-alive")
+
+(def ^:const auth-keep-alive-done-parameter "done=true")
```

Comment:
Changde comment in `waiter.auth.authentication/process-auth-keep-alive-request` to:
`ensure loop back to this endpoint terminates instead of continuously triggering re-authentication`

Reply:
Nitpick: Extracting constants gives them scope outside of the original functions from which they were extracted and as such they deserve their own comments instead of relying on future contributors to ascertain their meaning from their use within functions.